include/deal.II/lac/precondition.h

```diff
@@ -940,18 +937,6 @@ class PreconditionChebyshev : public Subscriptor
                const VectorType &src) const;
 
   /**
-   * Perform one step of the preconditioned Richardson iteration.
-   */
-  void step (VectorType       &dst,
-             const VectorType &src) const;
```

Comment:
I think this is a change that's probably unrelated to what you want to achieve with the current patch (extend `VectorTools::project`). Could this be separated into a different pull request? (I'm asking because I don't know why you want to remove the function...)